corona-warn-app/cwa-app-ios
# Feature/5409 Event Check In Info Screen

## Description
Implementation of the event checkin info screen. Unit test and first UI test available.

### How to test

- open the app and tab the "Check In" icon in the tab bar to open the info screen
- scroll down to the bottom and tap the button "Weitere Hinweise ..." to navigate to the data privacy note
- in case the user gives the consent (blue button "Einverstanden") the info screen will not be displayed any more. Reset the app to show the info screen again.

## Restrictions

- implemented placeholders for legal texts

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5409

## Screenshots

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-17 at 08 28 52](https://user-images.githubusercontent.com/7896005/111430878-5079b980-86fb-11eb-94c3-d0aceb15e19e.png)

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-17 at 08 29 01](https://user-images.githubusercontent.com/7896005/111430865-4bb50580-86fb-11eb-867e-30731f6ae5ca.png)

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-17 at 08 27 58](https://user-images.githubusercontent.com/7896005/111430898-58395e00-86fb-11eb-9aa4-01ffe6e778f2.png)

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-17 at 08 28 13](https://user-images.githubusercontent.com/7896005/111430885-540d4080-86fb-11eb-9fbf-ddbb72ba9ace.png)
